Source/inv.cpp

```diff
@@ -1776,39 +1772,37 @@ void CheckInvCut(int pnum, int mx, int my, bool automaticMove)
 						case PC_BARBARIAN:
 							PlaySFX(PS_WARR15, false);
 							break;
-

```

Comment:
Can we change the formatting standard to leave a blank line between cases? It is much harder to read without them IMHO :(

Reply:
Blank lines between cases are unusual and there's already the `break;` to visually separate the cases.